### DISCUSSION TITLE
knowledge/blog/set-up-front-end-development-environment-part-1/

### DISCUSSION THREAD

**Discussion by giscus** [TickLabVN/knowledges#17] [Q&A]
# knowledge/blog/set-up-front-end-development-environment-part-1/

Tổng hợp nội dung chia sẻ kiến thức trong quá trình nghiên cứu khoa học và phát triển dự án tại các phòng ban của TickLab.

https://ticklabvn.github.io/knowledge/blog/set-up-front-end-development-environment-part-1/

**Comment by amyranotamirror** [comment]
Dedicated work 🥂

**Comment by quannhg** [comment]
Looking forward to part 2
